haystack/backends/solr_backend.py

```diff
@@ -134,7 +134,7 @@ def search(self, query_string, **kwargs):
 
     def build_search_kwargs(self, query_string, sort_by=None, start_offset=0, end_offset=None,
                             fields='', highlight=False, facets=None,
-                            date_facets=None, query_facets=None,
+                            date_facets=None, query_facets=None, range_facets=None,
```

Comment:
Can you add this to the end of the kwargs for the method in case we run into someone's existing code not passing in kwargs by name?